rdt-app/src/main/java/io/ona/rdt_app/interactor/PatientRegisterFragmentInteractor.java

```diff
@@ -12,10 +12,8 @@
 import org.json.JSONObject;
 import org.smartregister.clientandeventmodel.Client;
 import org.smartregister.clientandeventmodel.Event;
-import org.smartregister.clientandeventmodel.Obs;
 import org.smartregister.domain.LocationProperty;
 import org.smartregister.domain.db.EventClient;
-import org.smartregister.domain.db.Obs;
```

Comment:
There is missing Obs Imports on this class

Reply:
Fixed.